README.md

```diff
@@ -61,6 +60,108 @@ yarn start
 
 Finally, we can navigate a browser window to `localhost:8080` to view generated data.
 
+### Running with Docker
+
+You can build and run sourcecred in a container to avoid installing dependencies
+on your host. First, build the container:
+
+```bash
+$ docker build -t sourcecred .
+```
+
+If you want to build and customize the `SOURCECRED_DIRECTORY`, you can
+set that as a `--build-arg`:
+
+```bash
+$ docker build --build-arg SOURCECRED_DEFAULT_DIRECTORY=/tmp/data -t sourcecred .
+```
+
+You will still need to export a GitHub token, and then provide it to the container
+when you run it. Notice that we are also binding port 8080 so we can
+view the web interface that will be opened up.  The only argument needed is a command to load the GitHub repository to generate the sourcecred for:
+
+```bash
+REPOSITORY=sfosc/sfosc
```

Comment:
technically, the argument isn't a repository, but a project spec. right now, the project can specify a repository (as in `sfosc/sfosc`) or a whole organization (as in `@sourcecred`). would be nice to show that both are possible.

Reply:
I'll add this now, although likely @Beanow will change it with the updated use.